java/common/common-task/src/main/java/sleeper/common/task/RunDataProcessingTasks.java

```diff
@@ -87,7 +122,6 @@ public interface TaskLauncher {
 
     public void run(QueueMessageCount.Client queueMessageCount) {
         long startTime = System.currentTimeMillis();
-        String sqsJobQueueUrl = instanceProperties.get(COMPACTION_JOB_QUEUE_URL);
         int maximumRunningTasks = instanceProperties.getInt(MAXIMUM_CONCURRENT_COMPACTION_TASKS);
```

Comment:
This property seems to be specific to compaction. Do we have a similar property for bulk export, or can we add one?